test/quantization/core/test_quantized_op.py

```diff
@@ -253,7 +253,14 @@ def test_qrelu(self, X):
                 }
             }
         ]
-        self._test_activation_function(X, 'relu', relu_test_configs)
+        devices = ["cpu", "cuda"] if TEST_CUDA else ["cpu"]
+        for device in devices:
+            if (device == 'cuda') and (torch.backends.quantized.engine == 'qnnpack'):
+                continue
+            X_tensor = list(copy.deepcopy(X))
+            X_tensor[0] = torch.from_numpy(X_tensor[0])
+            X_tensor[0] = X_tensor[0].to(device)
```

Comment:
should we add this in the `_test_activation_function` helper function itself? e.g. by passing an `devices` argument?

Reply:
I think we can add, since under such the case ((device == 'cuda') and (torch.backends.quantized.engine == 'qnnpack')), there always be an error when running _test_activation_function. @z-a-f

Reply:
does changing the `devices` to only include `cpu` when `torch.backends.quantized.engine == 'qnnpack'` resolve this issue?

Reply:
It should and we can save two lines...

Reply:
I'm going to move '(device == 'cuda') and (torch.backends.quantized.engine == 'qnnpack')' into _test_activation_function.